docs/getting-started/security-ui.asciidoc

```diff
@@ -22,204 +23,69 @@ image::images/search-bar.png[]
-=== Detection & Response dashboard
-
-The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.
-
-[role="screenshot"]
-image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]
-
-[float]
-[[detection-engine-ui]]
-=== Alerts page
-
-The Alerts page allows you to view and manage all alerts to monitor activity within your network. Refer to <<detection-engine-overview, Detections and Alerts>> for more information.
+<<detection-engine-overview, *Alerts*>>: View and manage alerts to monitor activity within your network.
 
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
-[float]
-=== Findings page
-
-The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+<<findings-page, *Findings*>>: Review your Kubernetes infrastructure’s performance on a range of security benchmarks.
 
 [role="screenshot"]
 image::cloud-native-security/images/findings-page.png[The Findings page]
 
-[float]
-[[rules-page]]
-=== Rules page
-
-The Rules page allows you to view and manage all detection rules. Refer to <<rules-ui-management, Manage detection rules>> for more information about prebuilt and custom rules.
+<<timelines-ui, *Timelines*>>: Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are responsive and allow you to share your findings among other team members.
 
 [role="screenshot"]
-image::detections/images/all-rules.png[Shows the Rules page]
+image::images/timeline-ui.png[Shows the Timeline page]
 
-[float]
-[[exceptions-page]]
-=== Exception lists page
+TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
 
-The Exception lists page allows you to view and manage all rule exceptions. Refer to <<detections-ui-exceptions, Rule exceptions and value lists>> for more information about rule exceptions.
+<<cases-overview, *Cases*>>: Open and track security issues.
 
 [role="screenshot"]
-image::detections/images/exceptions-page.png[Shows the Exception lists page]
-
-[float]
-[[hosts-ui]]
-=== Hosts page
+image::cases/images/cases-home-page.png[Shows the Cases page]
+*Explore*: Expand this section to view the following pages:
 
-The Hosts page provides key metrics for host-related security events and a set of interactive data tables. Refer to <<hosts-overview, Hosts page>> for more information.
+* <<hosts-overview, *Hosts*>>: View key metrics for host-related security events through graphs, charts, and interactive data tables.
```

Comment:
```suggestion
* <<hosts-overview, *Hosts*>>: View key metrics for host-related security events using graphs, charts, and interactive data tables.
```
My understanding is that "through" while often used like "by means of" is actually mostly for physical movement or the state of having finished with something.

Reply:
Can we also change "view" to "examine" here?

```suggestion
* <<hosts-overview, *Hosts*>>: Examine key metrics for host-related security events using graphs, charts, and interactive data tables.
```